proto/prysm/v1alpha1/BUILD.bazel

```diff
@@ -174,6 +174,11 @@ ssz_electra_objs = [
     "SignedConsolidation",
 ]
 
+ssz_fulu_objs = [
+    "DataColumnIdentifier",
```

Comment:
Same with `DataColumnIdentifier`, but feel free to merge it. I'm sure you will add `DataColumnIdentifier` in the subsequent PRs

Reply:
Actually I did the opposite in 83ffc531830b29ffc17e31f1521c29c65466dd70: I added everything needed at once.

Reply:
That is to say:
- `DataColumnIdentifier`,
- `DataColumnSidecar`,
- `DataColumnSidecarsByRangeRequest`, and
- `MetadataV2`